src/components/UserFeedbackComponent.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { Container, Header, Table, Pagination, Segment, Divider } from 'semantic-ui-react';
+import { Header, Table, Pagination, Segment, Divider } from 'semantic-ui-react';
```

Comment:
@geeeh please confirm what is the difference between this component and the one in `components/User/UserFeedbackComponent.jsx`. If there's no difference, then this should be deleted and use the one nested in `User` folder